changelog/_unreleased/2024-10-27-add-option-to-exclude-hidden-products-from-sitemap.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Add criteria filter types to sw-string-filter
```

Comment:
the title is probably wrong 😁

Reply:
Copy paste from the other PR, I'll fix it

Reply:
Fixed